i18n/management/commands/gather_i18n_strings.py

```diff
@@ -25,3 +26,11 @@ def handle(self, *args, **options):
                 outpath = os.path.abspath(os.path.join(source_dir, model.__name__ + ".json"))
                 with open(outpath, 'w') as outfile:
                     json.dump(strings, outfile, indent=4, sort_keys=True)
+
+        # Gather strings for translation in html files.
+        # For some reason makemessages won't create the locale/ directory but will create
+        # the subdirectories
+        subprocess.call(["mkdir", "locale/"])
```

Comment:
we can use [`os.mkdir`](https://docs.python.org/2/library/os.html#os.mkdir) for this

Reply:
Done.